Anishchenko_V/HW11/Task2.py

```diff
@@ -0,0 +1,14 @@
+day = input('Enter weekday number: ')
```

Comment:
good job
look at this version of code
d = {
    1: 'Monday',
    2: 'Tuesday',
    3: 'Wednesday',
    4: 'Thursday',
    5: 'Friday',
    6: 'Saturday',
    7: 'Sunday'
}

while True:
    try:
        i = int(input('Enter the day of the week: '))
    except ValueError:
        print('You did not enter a number!')
    else:
        print(d.get(i, 'There is no such day of the week!'))